### ISSUE TITLE
Unable to connect to Unifi Protect 2.1.1

### ISSUE THREAD

**Issue by johnwarne-gt** [ep1cman/unifi-protect-backup#41]
* Unifi Protect Backup version:
0.7.1
* Unifi Protect version:
2.1.1
* Python version:
* Operating System:
Proxmox 7.2-7 (Debian)
* Are you using a docker container or native?:
Docker container at [latest release 0.7.1](https://github.com/ep1cman/unifi-protect-backup/pkgs/container/unifi-protect-backup/24675669?tag=0.7.1)

### Description

Describe what you were trying to get done.
Tell us what happened, what went wrong, and what you expected to happen.

After updating from 1.x.x of Unifi Protect to 2.1.1. and a subsequent restart of my host that runs the Unifi Protect Backup container, I am no longer able to connect the two successfully. I created a new Protect role and user according to the updated readme, and when I then started the container I got a notification that the new user connected. However, it gets stuck in a loop, outputting the same error messages over and over, and uses lots of CPU.

### What I Did

```
Paste the command(s) you ran and the output.
If there was a crash, please include the traceback here.
```

Here are the container logs that repeat over and over.

```
2022-07-16 21:23:17 [DEBUG]:unifi_protect_backup.unifi_protect_backup:	Config:
2022-07-16 21:23:17 [DEBUG]:unifi_protect_backup.unifi_protect_backup:	  address='192.168.1.1'
2022-07-16 21:23:17 [DEBUG]:unifi_protect_backup.unifi_protect_backup:	  port=443
2022-07-16 21:23:17 [DEBUG]:unifi_protect_backup.unifi_protect_backup:	  username='unifi_protect_backup'
2022-07-16 21:23:17 [DEBUG]:unifi_protect_backup.unifi_protect_backup:	  password=REDACTED
2022-07-16 21:23:17 [DEBUG]:unifi_protect_backup.unifi_protect_backup:	  verify_ssl=False
2022-07-16 21:23:17 [DEBUG]:unifi_protect_backup.unifi_protect_backup:	  rclone_destination='local:/data'
2022-07-16 21:23:17 [DEBUG]:unifi_protect_backup.unifi_protect_backup:	  retention='7d'
2022-07-16 21:23:17 [DEBUG]:unifi_protect_backup.unifi_protect_backup:	  rclone_args=''
2022-07-16 21:23:17 [DEBUG]:unifi_protect_backup.unifi_protect_backup:	  ignore_cameras=()
2022-07-16 21:23:17 [DEBUG]:unifi_protect_backup.unifi_protect_backup:	  verbose=1
2022-07-16 21:23:17 [DEBUG]:unifi_protect_backup.unifi_protect_backup:	  detection_types=['person']
2022-07-16 21:23:17 [DEBUG]:unifi_protect_backup.unifi_protect_backup:	  file_structure_format='{camera_name}/{event.start:%Y-%m-%d}/{event.end:%Y-%m-%dT%H-%M-%S} {detection_type}.mp4'
2022-07-16 21:23:17 [INFO]:unifi_protect_backup.unifi_protect_backup:	Starting...
2022-07-16 21:23:17 [INFO]:unifi_protect_backup.unifi_protect_backup:	Checking rclone configuration...
2022-07-16 21:23:17 [DEBUG]:unifi_protect_backup.unifi_protect_backup:	rclone found: /usr/bin/rclone
2022-07-16 21:23:17 [DEBUG]:unifi_protect_backup.unifi_protect_backup:	ffprobe found: /usr/bin/ffprobe
2022-07-16 21:23:17 [INFO]:unifi_protect_backup.unifi_protect_backup:	Connecting to Unifi Protect...
Traceback (most recent call last):
  File "/usr/bin/unifi-protect-backup", line 8, in <module>
    sys.exit(main())
  File "/usr/lib/python3.9/site-packages/click/core.py", line 1137, in __call__
    return self.main(*args, **kwargs)
  File "/usr/lib/python3.9/site-packages/click/core.py", line 1062, in main
    rv = self.invoke(ctx)
  File "/usr/lib/python3.9/site-packages/click/core.py", line 1404, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/usr/lib/python3.9/site-packages/click/core.py", line 763, in invoke
    return __callback(*args, **kwargs)
  File "/usr/lib/python3.9/site-packages/unifi_protect_backup/cli.py", line 109, in main
    loop.run_until_complete(event_listener.start())
  File "/usr/lib/python3.9/asyncio/base_events.py", line 642, in run_until_complete
    return future.result()
  File "/usr/lib/python3.9/site-packages/unifi_protect_backup/unifi_protect_backup.py", line 288, in start
    await self._protect.update()
  File "/usr/lib/python3.9/site-packages/pyunifiprotect/api.py", line 565, in update
    self._bootstrap = await self.get_bootstrap()
  File "/usr/lib/python3.9/site-packages/pyunifiprotect/api.py", line 742, in get_bootstrap
    return Bootstrap.from_unifi_dict(**data, api=self)
  File "/usr/lib/python3.9/site-packages/pyunifiprotect/data/base.py", line 111, in from_unifi_dict
    data = cls.unifi_dict_to_dict(data)
  File "/usr/lib/python3.9/site-packages/pyunifiprotect/data/bootstrap.py", line 175, in unifi_dict_to_dict
    return super().unifi_dict_to_dict(data)
  File "/usr/lib/python3.9/site-packages/pyunifiprotect/data/base.py", line 343, in unifi_dict_to_dict
    data[key] = cls._clean_protect_obj_dict(data[key], unifi_dicts[key], api)
  File "/usr/lib/python3.9/site-packages/pyunifiprotect/data/base.py", line 288, in _clean_protect_obj_dict
    items[key] = cls._clean_protect_obj(value, klass, api)
  File "/usr/lib/python3.9/site-packages/pyunifiprotect/data/base.py", line 272, in _clean_protect_obj
    return klass.unifi_dict_to_dict(data=data)
  File "/usr/lib/python3.9/site-packages/pyunifiprotect/data/devices.py", line 746, in unifi_dict_to_dict
    return super().unifi_dict_to_dict(data)
  File "/usr/lib/python3.9/site-packages/pyunifiprotect/data/base.py", line 642, in unifi_dict_to_dict
    return super().unifi_dict_to_dict(data)
  File "/usr/lib/python3.9/site-packages/pyunifiprotect/data/base.py", line 333, in unifi_dict_to_dict
    data[key] = cls._clean_protect_obj(data[key], unifi_objs[key], api)
  File "/usr/lib/python3.9/site-packages/pyunifiprotect/data/base.py", line 272, in _clean_protect_obj
    return klass.unifi_dict_to_dict(data=data)
  File "/usr/lib/python3.9/site-packages/pyunifiprotect/data/base.py", line 326, in unifi_dict_to_dict
    data[key] = convert_unifi_data(data[key], cls.__fields__[key])
  File "/usr/lib/python3.9/site-packages/pyunifiprotect/utils.py", line 169, in convert_unifi_data
    value = [convert_unifi_data(v, field) for v in value]
  File "/usr/lib/python3.9/site-packages/pyunifiprotect/utils.py", line 169, in <listcomp>
    value = [convert_unifi_data(v, field) for v in value]
  File "/usr/lib/python3.9/site-packages/pyunifiprotect/utils.py", line 185, in convert_unifi_data
    value = field.type_(value)
  File "/usr/lib/python3.9/enum.py", line 384, in __call__
    return cls.__new__(cls, value)
  File "/usr/lib/python3.9/enum.py", line 702, in __new__
    raise ve_exc
ValueError: 'homekit' is not a valid VideoMode
2022-07-16 21:23:18 [ERROR]:asyncio             :	Unclosed client session
client_session: <aiohttp.client.ClientSession object at 0x7fdb27e315b0>
2022-07-16 21:23:18 [ERROR]:asyncio             :	Unclosed connector
connections: ['[(<aiohttp.client_proto.ResponseHandler object at 0x7fdb27e308e0>, 3931.857471964)]']
connector: <aiohttp.connector.TCPConnector object at 0x7fdb27e31850>
```

Any suggestions?

**Comment by ep1cman** [comment]
Hi,

Thanks for reporting this, looks like Ubiquiti have added some new video modes that are tripping up pyunifiprotect that this tool relies on. I will try to bump to the latest pyunifiprotect on the dev branch, please wait an hour or two for the containers to build and propagate then try using the ‘:dev’ tag of the container and see if that fixes it

**Comment by johnwarne** [comment]
Will do! I'll let you know how it goes once I've updated.

**Comment by ep1cman** [comment]
The dev container has now been pushed

**Comment by johnwarne-gt** [comment]
Working as expected on my end now!

**Comment by ep1cman** [comment]
Fix is now part of release v0.7.2
